sites/public/pages/eligibility/disability.tsx

```diff
@@ -44,14 +44,16 @@ const EligibilityDisability = () => {
   const disabilityValues = [
     {
       id: "disabilityNo",
-      value: "no",
+      value: "false",
       label: t("t.no"),
     },
     {
       id: "disabilityYes",
-      value: "yes",
+      value: "true",
       label: t("t.yes"),
     },
+    // Treat this case as true because we don't want to filter out any
+    // properties if the user doesn't designate a status.=
```

Comment:
nit: typo `=`

This comment might be more helpful on line 57 in `getFilterUrl()` in income.tsx, especially if it adds the context that we don't want to hide listings that aren't reserved for people with disabilities.

Reply:
Done.